.github/workflows/build_aws_scheduled.yml

```diff
@@ -41,17 +41,16 @@ jobs:
           role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
           role-duration-seconds: 3600
 
-      - name: Validate the Packer template
-        uses: hashicorp/packer-github-actions@master
+      - name: Setup packer
+        uses: hashicorp/setup-packer@main
```

Comment:
This has nothing to do with this task but `hashicorp/packer-github-actions` got deprecated so I needed to update it everywhere.